cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -55,8 +55,8 @@ def test_retire_service(appliance, setup_provider, context, order_catalog_item_i
 @pytest.mark.meta(blockers=[BZ(1544535, forced_streams=['5.9'])])
 @pytest.mark.parametrize('context', [ViaSSUI])
 @pytest.mark.parametrize('order_catalog_item_in_ops_ui', [['console_test']], indirect=True)
-@pytest.mark.uncollectif(lambda provider: provider.one_of(VMwareProvider) and
-                         provider.version >= 6.5,
+@pytest.mark.uncollectif(lambda provider: (provider.one_of(VMwareProvider) and
+                         provider.version >= 6.5) or provider.name == 'rhv_cfme_integration',
```

Comment:
**Required**
I don't think this is a good way to uncollect a test, since this is based only on what the provider name is.

Providers have 'excluded_test_flags' in their data, and I think uncollecting based on the inclusion of the `html5_console` flag in the yaml is a much better way of uncollecting this test.


```
@pytest.mark.uncollectif(lambda provider: (provider.one_of(VMwareProvider) and provider.version >= 6.5) or
                                                                     'html5_console' in provider.data.excluded_test_flags)
```

This could also be done with a provider mark that includes this flag in the provider filter. This module is marked at the top for provider parametrization, but you can separately decorate individual functions for unique generation:

```
@pytest.mark.provider(gen_func=providers,
                                      filters=[ProviderFilter(classes=[InfraProvider, CloudProvider],
                                                  required_fields=['provisioning'],
                                                  required_flags=['html5_console']])])
```


Lastly this could be done with test metadata specifying the required_flag, but we're trying to deprecate and phase out this implementation type.